docs/sources/administration/service-accounts/create-service-account.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Create a service account in Grafana
+menuTitle: Create a service account
+aliases: [docs/sources/administration/service-accounts/add-service-account-token.md]
+description: 'How to create a service account in Grafana'
+weight: 50
+keywords:
+  - Service accounts
+---
+
+# Create a service account in Grafana
+
+You can create a service account in the service account tab of the organization. To manage your computed workload with a user that is not tied to a user specifically.
+
+Consider the following restriction when you create a service account:
+
+- A service account is identified by its name, which is unique to the entire suite of organizations. The name is set upon creation of the service account
```

Comment:
The second sentence starts to get into the task steps. I would remove it, doesn't seem necessary. I think what you are saying is "When you create a service account, you enter a name." - in which case, I'd remove it.

Reply:
What is a 'suite of organizations'? Do you mean organizations within an instance of Grafana?

Reply:
i think we have mentioned both of these things but in different places. Specifically when running into the first step when naming.